libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -105,6 +118,13 @@ bool AP_Camera_Backend::take_picture()
     return false;
 }
 
+// take a picture.  returns true on success
```

Comment:
this comment looks out-of-date.  Perhaps copy the comment from the .h file.  It might be good to mention that the time_interval argument is in seconds.